packages/realm-server/package.json

```diff
@@ -91,5 +91,8 @@
   },
   "volta": {
     "extends": "../../package.json"
+  },
+  "dependencies": {
+    "@sentry/node": "^7.105.0"
```

Comment:
Should this be in `devDependencies` like everything else? I’m confused why that’s where things are.